doc/releases/nautilus.rst

```diff
@@ -0,0 +1,9 @@
+
+
+
+Upgrading from Luminous
+-----------------------
+
+* During the upgrade from luminous to nautilus, it will not be possible to create
+  a new OSD using a luminous ceph-osd daemon after the monitors have been
+  upgraded to nautilus.
```

Comment:
I don't find it clear why this happens once this PR is in. Mind explaining the reasoning a bit?

Reply:
The luminous MonClient::get_monmap_privately() function sends a MGetMonMap (?) message to the mon to fetch the monmap without authenticating.  That code has been removed from the mon now...

Reply:
(the only caller for get_monmap_privately() in luminous was ceph-osd during mkfs)